easybuild/easyconfigs/g/GlimmerHMM/GlimmerHMM-3.0.4-foss-2016b.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'MakeCp'
+
+name = 'GlimmerHMM'
+version = '3.0.4'
+
+homepage = 'https://ccb.jhu.edu/software/%(namelower)s'
+description = """GlimmerHMM is a new gene finder based on a Generalized Hidden Markov Model.
+ Although the gene finder conforms to the overall mathematical framework of a GHMM, additionally
+ it incorporates splice site models adapted from the GeneSplicer program and a decision tree adapted
+ from GlimmerM. It also utilizes Interpolated Markov Models for the coding and noncoding models."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['ftp://ccb.jhu.edu/pub/software/%(namelower)s']
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['43e321792b9f49a3d78154cbe8ddd1fb747774dccb9e5c62fbcc37c6d0650727']
+
+start_dir = 'sources'
+
+files_to_copy = [(['sources/%(namelower)s'], 'bin'), (['train/%s' % x for x in ['build1', 'build2',
+                   'build-icm', 'build-icm-noframe', 'erfapp', 'falsecomp', 'findsites', 'karlin',
```

Comment:
`'build-icm'` shouldbe aligned with `'build1'` on line above

better approach is probably to use a local variable for the list of things to copy from `train/`:

```python
train_files = ['build1', ... ]
files_to_copy = [(['sources/%(namelower)s'], 'bin'), (['train/%s' % x for x in train_files], 'bin'), 'trained_dir']
```

Are you sure all those `train` files need to be copied though, is that mentioned in the install guide?

Reply:
@boegel The readme.train only tells to run `make` in the `train` directory and all those files are created by the makefile. So I would just copy them all to the `bin` directory. The train files are only needed if one wants to train GlimmerHMM differently (already pre-trained, that's what trained_dir contains).